client/components/jetpack/daily-backup-status/status-card/backup-scheduled.jsx

```diff
@@ -35,16 +36,22 @@ const BackupScheduled = ( { lastBackupDate } ) => {
 
 	const lastBackupTime = lastBackupDate.format( 'LT' );
 
-	// Calculates the remaining hours for the next backup + 3 hours of safety margin
-	const DAY_HOURS = 24;
-	const hoursForNextBackup = DAY_HOURS - today.diff( lastBackupDate, 'hours' ) + 3;
+	const { date: nextBackupDate } = useNextBackupSchedule();
+
+	// Calculate the time difference for hours and minutes
+	const hoursForNextBackup = nextBackupDate.diff( today, 'hours' );
```

Comment:
I was testing this on Jetpack Cloud live branch and noted this error:

![image](https://github.com/user-attachments/assets/197ae0db-691c-4757-ac47-0976086585c4)

It seems nextBackupDate could be potentially null, so I'd need to add some guard checks here.